robots/cmd/perf-report-creator/output/vm/CREATE-pods-count/2022-10-10/data/results.json

```diff
@@ -0,0 +1,17 @@
+{
+  "StartDate": "2022-10-10",
```

Comment:
How can I tell this data is weekly?

Reply:
Should we consider a different path so we have room for expansion?  `../2022-10-10/data/results.json` makes it seem like we're tracking daily.

Reply:
@rthallisey I had a bug in there where the `NumberOfDays` was not being updated, I fixed it and pushed a commit.

But the output-dir flag can help here `-output-dir output/weekly` will give you a separate directory for weekly collection

Reply:
@rthallisey I have copied this from one of the perf jobs  `robots/cmd/perf-report-creator/test-build-log.txt` to make sure the unit test for regex match are finding the right values that file has the real data. Going further @dhiller mentioned that we should modify the performance job to write the output to a separate file. but unfortunately this is the only way to get historical data

Reply:
addressed the original question by separating this into two, one to scrap the results from the perf job and second to aggregate it into a weekly format